packages/starknet-snap/src/index.ts

```diff
@@ -237,7 +247,7 @@ export const onInstall: OnInstallHandler = async () => {
 
 export const onUpdate: OnUpdateHandler = async () => {
   const component = panel([
-    text('Features released with this update:'),
+    text('Features released with this update!:'),
```

Comment:
We don't need the exclamation here ( ! )

Before we allowList the next version we should update the onUpdate hook to include a note that the update will be cleaning out the deprecated Goerli network.

Reply:
Do you have the message, or we should ask Chaima to provide the message?

Reply:
resolved